packages/wrangler/src/sites.ts

```diff
@@ -434,7 +434,7 @@ function validateAssetKey(assetKey: string) {
  *
  * Primarily this involves converting Windows backslashes to forward slashes.
  */
-function urlSafe(filePath: string): string {
+export function urlSafe(filePath: string): string {
```

Comment:
same here, but yeah, nbd

Reply:
indeed. Can we pls keep track of this follow-up somewhere, for both this and prev comment 🙏 ?